test/e2e/utils/file.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+)
+
+func ReadFullFile(path string) ([]byte, error) {
+	fp, err := os.OpenFile(path, os.O_RDONLY, 0666)
+	if err != nil {
+		return nil, err
+	}
+
+	defer fp.Close()
+	return io.ReadAll(fp)
+}
```

Comment:
This method looks exactly the same as [io.ReadFile](https://pkg.go.dev/os#ReadFile).

Reply:
It's **os**.ReadFile. Thanks for pointing out!!!